src/plays/tic-tac-toe/Game.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from "react";
+import { toast, ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+import { FaTimes, FaRegCircle } from "react-icons/fa";
+import Icon from "./Icon";
+
+import "./Game.css";
+
+// Declaring an array to store the game state. Initializing it with 9 empty strings.
+const gameArray = new Array(9).fill("");
+
+const Game = () => {
+  let [isCross, setIsCross] = useState(true);
+  let [winMessage, setWinMessage] = useState("");
+  let count = 0;
+
+  // New Game
+  const PlayAgain = () => {
+    return (
+      <div className="center">

```

Comment:
Component shouldn't know about it's positioning, let's remove this div (finally button is just a button and components that use this button should decide how to place it)